src/braces.cpp

```diff
@@ -1103,6 +1104,13 @@ static void process_if_chain(chunk_t *br_start)
       {
          break;
       }
+
+      if (cpd.settings[UO_mod_full_brace_if_chain_only].b)
```

Comment:
We need that loop to go through the entire if-else chain. We hit this point when we see an 'else', and this is the sign that this is not a simple 'if(a) b;' statement, so we want to add braces.